doc/man3/EVP_MD_meth_new.pod

```diff
@@ -84,7 +84,15 @@ together.  The available flags are:
 
 =item EVP_MD_FLAG_ONESHOT
 
-This digest method can only handles one block of input.
+This digest method can only handle one block of input.
+
+=item EVP_MD_FLAG_XOF
+
+This digest method is an extensible-output function (XOF).
+
+=item EVP_MD_FLAG_DIGALGID_MASK
```

Comment:
I don't think this is actually supposed to be a flag, but a mask covering all possible EVP_MD_FLAG_DIGALGID_* flags. But I'm just guessing since it is unused anyway.

Reply:
I'm not sure how that would work since the value of this flag is identical to the DIGALGID_CUSTOM one. In the absence of more information shouldn't we leave it as is?

Reply:
I'm tempted to remove it from the docs.

Reply:
Done.